static/styles/portico/portico.css

```diff
@@ -680,6 +681,31 @@ input.text-error {
     margin-left: -3px;
 }
 
+.skip-link {
+    position: absolute;
+    top: -100px;
+    left: 5px;
+    overflow: hidden;
+    margin-left: 5px;
+    border-radius: 6px;
+    padding: 6px 10px;
+    cursor: pointer;
+    background-color: hsl(153, 32%, 55%);
+    color: hsl(0, 0%, 98%);
+    font-size: 18px;
+    font-weight: 400px;
+
+    &:focus {
+        z-index: 5000;
+        transition: ease-in-out 0.5s;
```

Comment:
* 0.5s is way too slow; use 0.2s.
* You have the wrong order for the `transition` shorthand notation, it should be `transition: <property> <duration> <timing-function>`.
* The transition should be specified for `.skip-link`, not `.skip-link:focus`, so that it works in both directions.